src/components/cveRecordSearchModule.vue

```diff
@@ -46,7 +45,7 @@ watch(
     if (route.query?.query){
       queryString.value = route.query.query.trim();
       validateQueryString();
-      if (!cveListSearchStore.showHelpText) {
+      if (!cveListSearchStore.isSeachButtonDisabled) {
```

Comment:
I think you could call the `validate()` function in this block. i think it's the same logic